### ISSUE TITLE
Adding Channel Variable into a channels.originate

### ISSUE THREAD

**Issue by mrlane** [skrusty/AsterNET.ARI#2]
When i try to put in a channel variable it doesnt seem to make it to asterisk. 
```cs
List<KeyValuePair<string,string>> bridgeID = new List<KeyValuePair<string,string>>(){
                    new KeyValuePair<string, string>("BRIDGEID", BRIDGEID)};

                    foreach(var element in bridgeID)
                    {
                        Console.WriteLine(element);
                    }

                    roboClient.Channels.Originate("SIP/200", "2011", "from-internal-custom", 6, null, null, null, null, bridgeID, newid, null);
```

**Comment by mrlane** [comment]
When i try to put in a channel variable it doesnt seem to make it to asterisk. 
```cs
List<KeyValuePair<string,string>> bridgeID = new List<KeyValuePair<string,string>>(){
                    new KeyValuePair<string, string>("BRIDGEID", BRIDGEID)};

                    foreach(var element in bridgeID)
                    {
                        Console.WriteLine(element);
                    }

                    roboClient.Channels.Originate("SIP/200", "2011", "from-internal-custom", 6, null, null, null, null, bridgeID, newid, null);
```

**Comment by skrusty** [comment]
Looks like an issue of rendering the body as JSON. Will need to do some testing and see where the template generator is failing on this.

**Comment by skrusty** [comment]
This should now be resolved in master branch as of pull request #5
